QUARKUS-6258.md

```diff
@@ -40,12 +47,16 @@ According to Mario, this is intentional, since they consider upstream lang4chain
 Ollama modules can be started on the laptop in dev mode. Jlama ones can be run as well.
 The features we want to cover are being used in integration-tests modules for OpenAI (Chats, Chat Memory, user/system messages, Function calling/Tools, Guardrails), RAG,. There is no coverage for WebSockets (only in codestarts) and for ChatModel (only in mocktests).
 
-### Impact on test suite
+## Impact on test suite
 We will focus on updating and running upstream tests from the Quarkiverse extensions against productised binaries.
 
 Primary focus will be on RHEL-based testing to ensure functionality of proposed features against productized bits. For the OpenShift side, we will explore the possibility of using the `@GitRepositoryQuarkusApplication` approach.
 
-As we aim for tech preview level for 3.27, we won't be focusing on full platform matrix coverage. We will be gaining experience to expand the coverage and platforms in the upcoming releases.d testing.
+As we aim for tech preview level for 3.27, we won't be focusing on full platform matrix coverage. We will be gaining experience to expand the coverage and platforms in the upcoming releases.
+
+## Impact on test suites for Full Support
+Since Quarkus AI may be moved to full support in its first release (Quarkus 3.27), we need to have a full support on all platforms. This means, that all tests should be in Quarkus Test suite, to guarantee running an all supported platforms.
+Additionally, we should add verification for this extension to Marete and startstop tests.
```

Comment:
Marete needs to be updated even for tech preview extensions. We test support levels of extensions.

As mentioned before, please clarify support level for extensions, and explicitly mention what needs to be done for acceptance testing.

What start-stop scenarios do you anticipate?

Let's be a bit more precise.

Reply:
Support level is described in the upstream issue, wouldn't that be enough?

> What start-stop scenarios do you anticipate?
No idea. We run start-stop tests for other supported extensions, maybe we should do it for these ones as well.